src/modal/modal.js

```diff
@@ -457,6 +457,12 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.multiMap', 'ui.bootstrap.sta
             close: '$close($value)',
             dismiss: '$dismiss($value)'
           });
+          Array.prototype.forEach.call(
+            Object.keys(modal.scope.$resolve),
+            function(property) {
+              content.attr(property, '$resolve.' + property);
```

Comment:
It should be something like
`content.attr(property.replace(/[A-Z]/g, function(c) { return '-' + c.toLowerCase(); }), '$resolve.' + property);`
to make it respect the angular "snake case" / dash-case for bindings with caps in their name (eg. mySuperDuperBinding -> my-super-duper-binding)